fdk/async_http/server.py

```diff
@@ -149,13 +151,20 @@ def serve(
         debug=debug,
     )
 
-    server_coroutine = loop.create_server(
-        server,
+    create_server_kwargs = dict(
         ssl=ssl,
         reuse_port=reuse_port,
         sock=sock,
         backlog=backlog,
-        start_serving=False
+    )
+
+    if constants.is_py37():
```

Comment:
testing-wise it would be nice to do things the same way, am testing 3.6 and 3.7 with this

Reply:
i think we can start it with start serving true, I forget on that long thread why we tried the alternative but I don't remember it making much of a difference (we were experimenting)

Reply:
we've used `start_serving`  in order to avoid races between FDK and Fn in terms of request reading/writing.